### ISSUE TITLE
Release of the name

### ISSUE THREAD

**Issue by ShalokShalom** [pszalanski/Diamond#1]
Hi there - it seems that this project is unmaintained since years, are you ready to close this repo, so Google hits and so on become more precise? Its for a new project, namely a Crystal <> Ruby transpiler. :smile:

**Comment by pszalanski** [comment]
Yeah, this was just an experimental side project, which I haven't touched for years. I'll gladly close it.
